src/compiler/ruby_generator.cc

```diff
@@ -160,12 +160,20 @@ grpc::string GetServices(const FileDescriptor* file) {
       return output;
     }
 
+    std::string package_name;
+
+    if (file->options().has_ruby_package()) {
```

Comment:
this is a slightly breaking change in that the new `grpc_ruby_plugin` will only work with `protobuf` 3.6 an onwards, correct? If so, can we make note of this in the PR description so that it gets in to the release notes. (I think that this "breakage" is ok because protoc and the plugin are packaged together).

Reply:
since 3.6 has been merged in, this shouldn't be a breaking change to the behavior of it. I will update the PR message appropriately

Reply:
updated PR description